gatsby-config.js

```diff
@@ -83,8 +83,6 @@ module.exports = {
         env: {
           production: {
             policy: [{ userAgent: '*' }],
-            sitemap: null,
-            host: null,
```

Comment:
This was removed because the build stage was failing and to reflect the `gatsby-plugin-robots-txt` documentation https://www.gatsbyjs.com/plugins/gatsby-plugin-robots-txt/#netlify which appears to exclude the sitemap and host properties.